package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "configstore",
-  "version": "1.4.0",
+  "version": "2.0.0",
```

Comment:
Sure, will revert that. Thought I could bump it because you were talking about a major update. Sorry for that :grimacing: